### ISSUE TITLE
ripple did not appear

### ISSUE THREAD

**Issue by psyst8** [jatchili/minimalist-ripple-client#12]
I sent 20 ripple to an account and it did not show up at all

It has left my account but not entered into the minimalist ripple client,

when checking account status is says account not found!

**Comment by Emil113** [comment]
What's your public address that you sent the ripple to?

**Comment by psyst8** [comment]
The public address is


rs4GYnsaLMNftxJRmbwiHv2BbSmGsYBumF

________________________________

**Comment by psyst8** [comment]
this is from the other wallet


21/12/2017 06:06 PM     Send    -20.16732073
rs4GYnsaLMNftxJRmbwiHv2BbSmGsYBumF
        Complete



________________________________

**Comment by Emil113** [comment]
Yeah, something must not have gone through correctly.  https://xrpcharts.ripple.com/#/graph/rs4GYnsaLMNftxJRmbwiHv2BbSmGsYBumF shows that the account is not found.  What did the exchange say on the transaction?  Is it complete?

When I did it it took less than a few minutes to see the funds show up so check again if the account you created with this software is correct.  Also make sure you DO NOT use safari web browser as that will generate invalid keys.  I used Chrome and it worked perfectly

**Comment by psyst8** [comment]
it says its complete on the exchange




________________________________

**Comment by Emil113** [comment]
how did you create the key? On what browser? Was it on a computer offline or online?

**Comment by psyst8** [comment]
online computer using safari i believe :\

________________________________

**Comment by Emil113** [comment]
yeah that was the problem, safari can't generate correct keys so you need to create a new set of keys using something like Chrome.

**Comment by psyst8** [comment]
and the other coins are gone?

________________________________

**Comment by Emil113** [comment]
I'm not 100% sure.  My guess is yes but you can open a support ticket with the transition on the exchange you used and see if they can see where they went and credit your account back.

**Comment by psyst8** [comment]
cheers, i'll try that, thanks

________________________________

**Comment by Cherikoff** [comment]
I used Chrome but have had the same problem. Any other clues to get the minimalist interface working when the key it generates shows No account found?

**Comment by psyst8** [comment]
So apparently the account needs at least 20xrp to be activated.

I used safari and was able to get the coins back from coinspot as the account created via minimalist was invalid.

Sent from my iPhone

On 16 Jan 2018, at 10:03 am, Cherikoff <notifications@github.com<mailto:notifications@github.com>> wrote:


I used Chrome but have had the same problem. Any other clues to get the minimalist interface working when the key it generates shows No account found?

**Comment by Cherikoff** [comment]
Thanks psyst8. I logged a ticket with Coinspot. I tranferred 30XRP to be sure but the public key must have been wrong. No sign of the coins on the blockchain.
